src/contact/parallel/bvh_contact_manager.cc

```diff
@@ -97,11 +97,17 @@ struct ArborXCallback
       //
       entry.local_index = myFace.local_id();
       entry.node        = false;
-      resa_vec.push_back(entry);
+#pragma omp critical
```

Comment:
This won't work with CUDA, but I think it's good for now. We should probably add support in BVH for kokkos views and add to one of those

Reply:
Agreed.
- I do not know if `Kokkos::View` allows simple `push back` and in an atomic sense.
- There is a `Kokkos::Vector` with `push_back`. It is not clear to me whether it would solve the critical issue.